components/conversations/Comment.js

```diff
@@ -65,13 +65,13 @@ const AdminActionButtons = ({ comment, canEdit, canDelete, isConversationRoot, o
     <React.Fragment>
       {/** Buttons */}
       {canEdit && (
-        <CommentBtn onClick={onEdit} ml={2}>
+        <CommentBtn onClick={onEdit} ml={2} data-cy="Comment-button">
```

Comment:
We should be a bit more explicit on the name here

```suggestion
        <CommentBtn onClick={onEdit} ml={2} data-cy="edit-comment-button">
```